lib/gui.py

```diff
@@ -63,11 +75,39 @@ def on_submit():
             # Set the label text to the password generated
             password = generator.gen(int(user_input.get()))
             password_label.config(text=f"Password generated: {password}")
-            copy_button = tk.Button(password_generator_window, text='Copy to clipboard',
-                                    command=lambda: copy_to_clipboard(password))
-            copy_button.pack()
-        else:
-            password_label.config(text="Invalid input provided!")  # Update the label text
+            global copy_button
+            copy_button = tk.Button(
+                password_generator_window,
+                text="Copy",
+                command=lambda: copy_to_clipboard(password),
+            )
+            copy_button.grid(row=3, column=2)
+            submit_button.grid(row=3, column=0)
 
-    submit_button = tk.Button(password_generator_window, text="Submit", command=on_submit)
-    submit_button.pack()
+        else:
+            password_label.config(text="Invalid input provided!")
+            # We shouldn't be able to copy if input was incorrect
+            copy_button.destroy()
+            submit_button.grid(row=3, column=1)
+
+    submit_button = tk.Button(
+        password_generator_window, text="Submit", command=on_submit
+    )
+    submit_button.grid(row=3, column=1)
+
+
+def main_window_generator():
+    root = tk.Tk()
+    x = root.winfo_screenwidth()
```

Comment:
Was the intention to make it full-screen? (Btw, I don't think these variables are used anywhere)

Reply:
they're not used yet. I wanted to use them for creating something at least slightly responsible later.

Reply:
sure